packages/experimental/TabList/src/Tab/Tab.types.ts

```diff
@@ -5,87 +5,60 @@ import type { IViewProps } from '@fluentui-react-native/adapters';
 import type { IconPropsV1 as IconProps } from '@fluentui-react-native/icon';
 import type { IFocusable, PressableState, PressablePropsExtended } from '@fluentui-react-native/interactive-hooks';
 import type { TextProps } from '@fluentui-react-native/text';
-import type { FontTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import type {
+  FontTokens,
+  IBackgroundColorTokens,
+  IBorderTokens,
+  IForegroundColorTokens,
+  LayoutTokens,
+} from '@fluentui-react-native/tokens';
+
+import type { TabIndicatorProps } from '../TabIndicator/TabIndicator.types';
 
 export const tabName = 'Tab';
 
-export interface TabTokens extends FontTokens, IBorderTokens {
+export interface TabTokens extends FontTokens, IBorderTokens, IForegroundColorTokens, IBackgroundColorTokens, LayoutTokens {
+  contentMarginHorizontal?: number;
+
+  flexDirection?: ViewStyle['flexDirection'];
   /**
    * The indicator color.
    */
-  indicatorColor?: string;
+  indicatorColor?: ColorValue;
 
-  /**
-   * The opacity of the Tab.
-   */
-  tabsItemOpacity?: number;
+  indicatorInset?: number;
```

Comment:
Might be helpful to have a comment here explaining what the indicatorInset is?

Reply:
+ 1 I wasn't quite sure

Reply:
Went ahead and added comments to each TabToken prop to clarify what is what.